multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/model/SupportedParameters.java

```diff
@@ -145,6 +145,7 @@ public class SupportedParameters {
     public static final String DEFAULT_XS_APP_NAME = "default-xsappname";
     public static final String TYPE = "type";
     public static final String SKIP_SERVICE_UPDATES = "skip-service-updates";
+    public static final String FAIL_ON_SERVICE_UPDATE = "fail-on-service-update";
```

Comment:
This should be added to the RESOURCE_PARAMETERS structure in this class, from the latest changes on master. That way this parameter won't be shown as a warning

Reply:
done